ui/app/adt/views/wardList.html

```diff
@@ -7,7 +7,7 @@
         <table class="table ward-details-content" ng-if="tableDetails.length > 0">
             <thead>
                 <tr>
-                    <th ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes' "ng-class="{'bed-detail-column' : $index == 0}">{{heading | titleTranslate}}</th>
+                    <th ng-repeat="heading in tableHeadings" ng-if="heading != 'hiddenAttributes' "ng-class="{'bed-detail-column' : $index == 0}">{{heading | translate}}</th>
```

Comment:
@PoojaSastry-TW can you mention the reason why we have changed from "titleTranslate" to "translate" ?